### DISCUSSION TITLE
Verifying document changes before merge

### DISCUSSION THREAD

**Discussion by ashvina** [apache/incubator-xtable#238] [Q&A] [answered]
Hello everyone,
I have updated the docs with a new tutorial for OneTable. Since I am new to documentation, I would appreciate some guidance on how to check the changes before I submit a PR. I guess there are no automated tests for this, so I need to do it manually. What are the best practices for deploying locally and testing? Thank you for your assistance.

**Comment by the-other-tim-brown** [comment, marked as answer]
The steps for running the docs website locally are captured in the [readme](https://github.com/onetable-io/onetable/tree/main/website). There will a check that npm can properly build the changes ([workflow](https://github.com/onetable-io/onetable/blob/main/.github/workflows/site-build.yml)) but nothing that verifies that the changes are aesthetically pleasing. I usually just take these PRs and view them on my local. Screenshots of the output would be nice in the PRs themselves.
